packages/bpk-component-chip-group/src/BpkMultiSelectChipGroup.tsx

```diff
@@ -177,34 +214,75 @@ const RailChipGroup = ({
   return (
     <>
       <BpkBreakpoint query={BREAKPOINTS.ABOVE_TABLET}>
-        <Nudger position={POSITION.leading} ariaLabel={leadingNudgerLabel} chipStyle={chipStyle} scrollContainerRef={scrollContainerRef} />
+        <Nudger
+          position={POSITION.leading}
+          ariaLabel={leadingNudgerLabel}
+          chipStyle={chipStyle}
+          scrollContainerRef={scrollContainerRef}
+        />
       </BpkBreakpoint>
-      {stickyChip &&
+      {stickyChip && (
         <div className={stickyChipContainerClassnames}>
           <BpkBreakpoint query={BREAKPOINTS.ABOVE_TABLET}>
-            {(isDesktop) =>
-              <Chip chipItem={({
-                role: 'button',
-                component: isDesktop ? CHIP_COMPONENT.selectable : CHIP_COMPONENT.icon,
-                leadingAccessoryView: <FilterIconSm />,
-                ...stickyChip,
-              })} chipStyle={chipStyle} ariaMultiselectable={ariaMultiselectable} chipIndex={-1} />
-            }
+            {(isDesktop) => {
+              const shouldHideStickyText = !isDesktop && !isAtStart;
```

Comment:
Consider moving this code and `useEffect` to a separate component for better readability